### PR TITLE
Initialize View Start Time During View Registration

### PR DESCRIPTION
**Pull request by ian-mi**
Fixes https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/issues/266

### CODE REVIEW

**Comment by ian-mi** [comment]
@nilebox

File: stats/view/worker.go [hunk 1]
```diff
@@ -333,20 +334,17 @@ func (w *worker) unregisterView(v *viewInternal) {
...
 	}
 }
 
-func (w *worker) reportView(v *viewInternal, now time.Time) {
+func (w *worker) reportView(v *viewInternal) {
 	if !v.isSubscribed() {
 		return
 	}
 	rows := v.collectedRows()
-	_, ok := w.startTimes[v]
```
Comment (nilebox) [inline stats/view/worker.go:346]: There is another place with the same code which can be deleted: https://github.com/census-instrumentation/opencensus-go/blob/1901b56b9515b0c34f5d25a5bce982dfc543d64b/stats/view/worker.go#L376-L379

File: stats/view/worker.go [hunk 2]
```diff
@@ -333,20 +334,17 @@ func (w *worker) unregisterView(v *viewInternal) {
 	w.mu.Lock()
 	defer w.mu.Unlock()
 	delete(w.views, v.view.Name)
+	delete(w.startTimes, v)
```
Comment (nilebox) [inline stats/view/worker.go:337]: 👍

File: stats/view/worker.go (hunk 1 above)
Comment (ian-mi) [reply stats/view/worker.go:346]: Good catch!

**Pull request merged**